### ISSUE TITLE
Facehuggers Entering Vehicles

### ISSUE THREAD

**Issue by Accoint** [cmss13-devs/cmss13#5365]
### Testmerges

#5298 #5113 #5285 #5153 #5086 #5337 #5280 #5278 #5122 #5359 #5336 #5226

### Description of the bug

Facehuggers cannot enter vehicles normally.

Facehuggers can enter vehicles by being dragged by another xeno, and them entering.

### What's the difference with what should have happened?

Facehuggers should be able to enter a vehicle.
Or.
Facehuggers should not be able to be dragged into the vehicle.

### How do we reproduce this bug?

1. Attempt to enter vehicle as facehugger.
Alternatively.
1. Get a xenomorph to drag the facehugger.
2. Get the xenomorph to click on the vehicle, bringing the dragged facehugger inside.

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by MistChristmas** [comment]
Yeah they get the attack_alien larba treatment. But probably should be able to interact with vehicle doors.
